sdk/eventgrid/Azure.Messaging.EventGrid.Namespaces/perf/Azure.Messaging.EventGridV2.Perf.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We should just zap the `perf` and `stress` directories until we're actually going to use them.